src/main/java/nl/b3p/tailormap/api/controller/UniqueValuesController.java

```diff
@@ -140,14 +141,15 @@ private UniqueValuesResponse getUniqueValues(
                                 null);
             } else {
                 // #2 or use a Function to get the unique values
-                // this is the recommended way, uses SLQ "distinct"
+                // this is the recommended way, uses SQL "distinct"
                 logger.trace("Using geotools unique collection function to get unique values");
                 Function unique = ff.function("Collection_Unique", ff.property(attributeName));
                 Object o = unique.evaluate(fs.getFeatures(q));
                 if (o instanceof Set) {
                     @SuppressWarnings("unchecked")
                     Set<Object> uniqueValues = (Set<Object>) o;
```

Comment:
Why not use a SortedSet instead

Reply:
I guess this works too, but it needs to be a `List` for the `UniqueValuesResponse` eventually. As we get a `Set` from the `Function` it only needs to be sorted, creating a new `TreeSet` again isn't necessary.

```
                    SortedSet<Object> uniqueValues = new TreeSet<>((Set<Object>) o);
                    uniqueValuesResponse.setValues(List.of(uniqueValues));
```

But the difference probably isn't big, we have other problems should we request unique values of millions of uids where this would make a difference.

Reply:
Misunderstood your suggestion, with `uniqueValues: true` in the model and using `SortedSet` in the Swagger class works!